### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Tuesday, 13th of September 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by miao0miao** [flatcar/Flatcar#851] [Flatcar Office Hours]
## Agenda for the Flatcar Office Hours on Tuesday, 13th of September 2022  17:30 CEST / 11:30am EDT / 8:30am PDT

* Zoom link:  https://us06web.zoom.us/j/84336116610
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: [link](https://www.youtube.com/watch?v=JqFXLShMW7k)
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- past events:  FrOSCon , https://www.froscon.org/en/
- upcoming: Hacktoberfest is coming and we're participating! See https://hacktoberfest.com/participation/ for more information. 
- @tormath1 will present the OpenStack "infra" - how to test and debug Flatcar on it and finally introduce the final step: adding OpenStack in the CI pipeline.
- Announcement - "flatcar-linux" GitHub organisation will be renamed to "flatcar" ([tracking issue](https://github.com/flatcar-linux/Flatcar/issues/836))
   - We don't expect immediate fallout as redirects will be in place for `flatcar-linux/...` repo URLs to `flatcar/...`.
      Long term, people should update their references though.
   - @invidian will join us and elaborate on porting work for go module dependencies (e.g. [config-transpiler](https://github.com/flatcar-linux/container-linux-config-transpiler)) to use the new org, as well as on how you can check if you're affected
   - @johananl will be around to elaborate on ClusterAPI PRs which update Flatcar CAPI support for the new repo URLs / go modules

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by tormath1** [comment]
I can briefly present the OpenStack "infra", how to test and debug Flatcar on it and finally introduce the final step: adding OpenStack in the CI pipeline.
Related items:
- https://github.com/flatcar-linux/Flatcar/issues/785
- https://github.com/flatcar-linux/mantle/pull/350
- https://github.com/flatcar-linux/mantle/pull/349

Slides: [openstack-flatcar.pdf](https://github.com/flatcar-linux/Flatcar/files/9558687/openstack-flatcar.pdf)

**Comment by t-lo** [comment]
Added more detail on the org name change.
